crates/bifrost/src/providers/local_loglet/log_store.rs

```diff
@@ -133,14 +133,13 @@ impl RocksDbLogStore {
     }
 }
 
-fn db_options(options: &LocalLogletOptions) -> rocksdb::Options {
+fn db_options() -> rocksdb::Options {
     let mut opts = rocksdb::Options::default();
 
-    // enable atomic flushes to not persist inconsistent data in case WAL
-    // is disabled
-    if options.rocksdb.rocksdb_disable_wal() {
-        opts.set_atomic_flush(true);
-    }
+    // enable atomic flushes
+    // if wal is disabled, this ensure we do not persist inconsistent data
+    // if wal is enabled, this ensures that flushing either cf flushes both
+    opts.set_atomic_flush(true);
```

Comment:
I would appreciate if you leave a comment with the details on why we actually want to do this (avoid lingering wal files eating up disk space on lightly loaded nodes)